package/ci/appveyor-desktop.bat

```diff
@@ -65,13 +65,16 @@ cd .. && cd ..
 
 rem Build. MAGNUM_BUILD_GL_TESTS is enabled just to be sure, it should not be
 rem needed by any plugin.
+rem OpenCL is disabled on MSVC 2017 to get some test coverage for
+rem BasisImageConverter without OpenCL support compiled in.
 mkdir build && cd build || exit /b
 cmake .. ^
     -DCMAKE_BUILD_TYPE=Debug ^
     -DCMAKE_INSTALL_PREFIX=%APPVEYOR_BUILD_FOLDER%/deps ^
     -DCMAKE_PREFIX_PATH=%APPVEYOR_BUILD_FOLDER%/openal;%APPVEYOR_BUILD_FOLDER%/libwebp;%APPVEYOR_BUILD_FOLDER%/devil;C:/Tools/vcpkg/installed/x64-windows ^
     -DMAGNUM_WITH_ASSIMPIMPORTER=%EXCEPT_MSVC2015% ^
     -DMAGNUM_WITH_ASTCIMPORTER=ON ^
+    -DCMAKE_DISABLE_FIND_PACKAGE_OpenCL=%EXCEPT_MSVC2015% ^
```

Comment:
This disables it for all builds except MSVC 2015 tho, not for 2017 only.

Also 2017 gets the 1.15 version now, which doesn't have OpenCL yet, so try to disable it on 2019 instead?